git/test/test_submodule.py

```diff
@@ -802,6 +802,24 @@ def assert_exists(sm, value=True):
         # end for each dry-run mode
 
     @with_rw_directory
+    def test_remove_norefs(self, rwdir):
+        parent = git.Repo.init(os.path.join(rwdir, 'parent'))
+        sm_name = 'mymodules/myname'
+        sm = parent.create_submodule(sm_name, sm_name, url=self._small_repo_url())
+        assert sm.exists()
+
+        parent.index.commit("Added submodule")
+
+        assert sm.repo is parent  # yoh was surprised since expected sm repo!!
```

Comment:
and I am not sure why that is the case -- I was expecting submodules repo to be the repo of the submodule ... but I guess it can make sense to refer to parent which contains the submodule, but then is there an attribute for submodules repo (if present)?